test/index_tests.js

```diff
@@ -427,6 +428,18 @@ it('WHEN node environment is ignored but always serve docs is enabled THEN it sh
   });
 });
 
+it('WHEN specOutputFileBehavior is set to PRESERVE THEN it should load existing spec file', () => {
+  const app = express();
+  const specOutputPath = './test/outputs/test_spec.json';
```

Comment:
Cannot find the file, I suppose it should exist before we execute generator code and some data should remain the same here even after REST API calls. But no file, no REST API calls are made in the test, or have I missed something?

Reply:
Ha! The file name was on .gitignore. Already changed to `example_spec.json`.